.github/workflows/ci.yml

```diff
@@ -35,3 +35,11 @@ jobs:
       - name: Run tests
         run: |
           pytest tests/.
+  docker:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Build Docker image
+        run: docker build -t my-app:latest .
```

Comment:
it is a CD step, you should trigger such things when you deliver project to the user. CI is responsible for code quality and that the code delivered to main branch is of appropriate quality. please create a separate file with that. CDs are triggered usually on merge to main